lib/drivers/common/prepare.js

```diff
@@ -47,13 +47,19 @@ function spawn(dir, env, cmd, callback) {
   var options = {
     cwd: dir,
     env: _.extend({}, process.env, env),
-    stdio: 'inherit',
+    stdio: debug.enabled ? ['ignore', 'pipe', 'pipe'] : 'ignore',
```

Comment:
I tested and install of no-such-package, and an npm rebuild (there are compiler warnings for agent on stderr), and I think it gives adequate information in both cases.